tests/integration_test_data_endpoint.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+import re
+
+from syncano.models import Class, DataEndpoint, Object, ResponseTemplate
+from tests.integration_test import InstanceMixin, IntegrationTest
+
+
+class DataEndpointTest(InstanceMixin, IntegrationTest):
+
+    schema = [
+        {
+            'name': 'title',
+            'type': 'string',
+            'order_index': True,
+            'filter_index': True
+
+        }
+    ]
+
+    template_content = '''
```

Comment:
Each instance - after creation has its own templates: `objects_csv` and `html_table_list` - if I am remember this. You can rely on this - and do no create a template here.

CSV - will return file, but html table - will return html :)